src/client/src/rt-theme/colors.js

```diff
@@ -0,0 +1,203 @@
+import _ from 'lodash';
```

Comment:
Can you make this a TS file, please. Like the rest of the application

Reply:
Let's pair on this.